questionnaires/esm_epi_mod_1/esm_epi_mod_1_armt.json

```diff
@@ -0,0 +1,2186 @@
+                "label": "Relatives not living with you "
+            },
+            {
+                "code": "4",
+                "label": "House/room mates "
+            },
+            {
+                "code": "5",
+                "label": "Friends "
+            },
+            {
+                "code": "6",
+                "label": "Classmates/colleagues "
+            },
+            {
+                "code": "7",
+                "label": "Caregiver(s) "
+            },
+            {
+                "code": "8",
+                "label": "Strangers/others"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "esm_social_people_46f3a0",
+        "form_name": "esm_epi_mod_1",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "How many people am I with?",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "1"
+            },
+            {
+                "code": "2",
+                "label": "2"
+            },
+            {
+                "code": "3",
+                "label": "3-10"
+            },
+            {
+                "code": "4",
+                "label": "over 10"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[esm_social_interact(1)] = \"1\" or [esm_social_interact(2)] = \"1\" or [esm_social_interact(3)] = \"1\" or [esm_social_interact(4)] = \"1\" or [esm_social_interact(5)] = \"1\" or [esm_social_interact(6)] = \"1\" or [esm_social_interact(7)] = \"1\" or [esm_social_interact(8)] = \"1\"",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": "responses['esm_social_interact'] == 1 || responses['esm_social_interact'] == 2 || responses['esm_social_interact'] == 3 || responses['esm_social_interact'] == 4 || responses['esm_social_interact'] == 5 || responses['esm_social_interact'] == 6 || responses['esm_social_interact'] == 7 || responses['esm_social_interact'] == 8"
+    },
+    {
+        "field_name": "esm_social_doing_7a5c14",
+        "form_name": "esm_epi_mod_1",
+        "section_header": "",
+        "field_type": "range",
+        "field_label": "We are doing something together",
```

Comment:
why is this a range? This is like a `yes` or `no` question.

Reply:
guess this is part of the prior ESM, guess it's a degree of doing things together :wink: